js/aax.js

```diff
@@ -14,7 +14,7 @@ module.exports = class aax extends Exchange {
         return this.deepExtend (super.describe (), {
             'id': 'aax',
             'name': 'AAX',
-            'countries': [ 'MT' ], // Malta
+            'countries': ['MT'], // Malta
```

Comment:
Thanks for a quick reaction! Also, if possible, please, revert the whitespace removal from lines 17, 978, 1342, 1378, 1536, 2089 – we want to keep the codestyling consistent across the exchanges.

Reply:
Sure, I'll fix it! It was actually the linter that changed these.
Would it make sense to make the rule in eslint to?
` "array-bracket-spacing": "on",`

Reply:
Yep, that would be awesome. Please, note, that we have multiple .eslintrc files in the repo, the base code is more relaxed than the derived exchange class files. I mostly mean the one here: https://github.com/ccxt/ccxt/blob/master/js/.eslintrc

Reply:
Cool, I'll do that in a separate PR for only the exchange class files.

Reply:
@pcriadoperez thank you!